models/public/anti-spoof-mn3/model.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Antispoofing model "anti-spoofing-mn3" based on the mobilenetv3 originally trained on PyTorch*
```

Comment:
Use [script](https://github.com/openvinotoolkit/open_model_zoo/tree/develop/ci/documentation_updater) to update description.